src/Tests/ObjectLifetimeTests/ObjectLifetimeTests.Tests/ObjectLifetimeTests.Tests.csproj

```diff
@@ -0,0 +1,29 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
```

Comment:
Is this file used?  If not, do you think it would be easy to just merge the other 2 folders (test and ObjectLifetimeTests.Lifted)

Reply:
Unused, removed and merged folders.